File: java/src/main/java/heritage/Personne.java
package heritage;

public class Personne {
    protected String nom;
    protected int age;

    public Personne(String nom, int age) {
        this.nom = nom;
        this.age = age;
    }

    public String getNom() {
        return nom;
    }

    public int getAge() {
        return age;
    }

    @Override
    public String toString() {
        return "Personne: " + nom + ", " + age + " ans";
    }
}

File: python/heritage/src/personne.py
class Personne:
    def __init__(self, nom: str, age: int):
        self.nom = nom
        self.age = age

    def __str__(self):
        return f"Personne: {self.nom}, {self.age} ans"

    def __repr__(self):
        return f"Personne({self.nom!r}, {self.age!r})"
